data/darktableconfig.xml.in

```diff
@@ -2009,6 +2009,20 @@
     <shortdescription>Geotagging search URL</shortdescription>
     <longdescription>this can be changed when the default OpenStreetMap search URL is broken</longdescription>
   </dtconfig>
+  <dtconfig prefs="gui" section="darkroom">
+    <name>plugins/darkroom/temperature/colored_sliders</name>
+    <type>bool</type>
+    <default>true</default>
```

Comment:
I would keep the default to false.

Reply:
Hmm... My only UX tester except me is my wife and according to her it should be default true, bacause it deals with color therefore needs to be in color :stuck_out_tongue_closed_eyes:

Plus I found users like @moy mentioning that they loved the color but found temperature slider a bit confusing (that's why I introduced `blackbody_is_confusing`) option.

Let me ask others: https://discuss.pixls.us/t/darktable-ui-ux-white-balance-module-poll/16626

Reply:
Well my point is that it was disabled because found to be confusing, see 72d8dd27d5c49c656218efc914be087c1f65138e. So better not have it back by default.

Reply:
I studied those cases - most were caused by blackbody radiation instead of more "well-known" lightroom-like one. And in my opinion the confusion was nowhere to be alleviated (no manual, no tooltip on it) to help guide the user. Most people were very happy about coloured tint slider, most people don't really touch R/G/B sliders so the painpoint (IMHO) is in temperature slider.

So far it seems that colored sliders are overall seen in positive light. I do plan on fixing it a bit more (CYGM/RGBE sensors) and adding better tooltips for temperature slider.

How about - if you agree, let's add it (once it's done) as "disabled by default" in 3.0.x tag, but "enabled by default" on 3.x tag?

Reply:
Ok, no strong opinion on my side, so let's keep it default for now.

Reply:
And BTW thanks for the explanation.

Reply:
Can we keep it in "WIP" state for a bit longer? guille2306 (Guillermo Rozas) addressed a bit of a problem that I sorta agree with, but is problematic: RGB sliders can be a bit confusing given that they don't "make image more blue" but rather change coefficient of blue inrelation to other coefficients... So I think I'll try to make something similar to what GIMP does, that is - update other sliders colors while updating current one value.

Reply:
WIP label is no longer needed. As I mentionedin convo elsewhere, the functionality in this PR is complete. further changes will be subject to other PRs :)